Source/Core/VideoCommon/CustomTextures.cpp

```diff
@@ -140,10 +140,11 @@ void HiresTexture::Update()
   }
 }
 
-void HiresTexture::Prefetch()
+void CustomTexture::Prefetch()
 {
   Common::SetCurrentThreadName("Prefetcher");
 
+  //TODO Fix this whole thing - Consensus reached at IRC logs MAY-2017
```

Comment:
Fix *what* exactly? Also not everybody has IRC logs, and just saying May isn't clear enough as there are 31 days in May... May should also not be all caps.